app/registries/foreman/access_permissions.rb

```diff
@@ -132,7 +132,7 @@
                                         :"api/v2/template_combinations" => [:destroy],
                                       }
     map.permission :deploy_provisioning_templates,  {:provisioning_templates => [:build_pxe_default],
-                                        :"api/v2/provisioning_templates" => [:build_pxe_default],
+                                        :"api/v2/provisioning_templates" => [:build_pxe_default, :global_registration],
```

Comment:
I don't think this is the right permission, we should either add it to existing `:create_hosts` or add entirely new one called `:register_hosts`. I think I prefer the first one.